make/istio.mk

```diff
@@ -19,15 +26,32 @@ $(ISTIOCTL):
 .PHONY: istioctl
 istioctl: $(ISTIOCTL) ## Download istioctl locally if necessary.
 
-.PHONY: istio-install
-istio-install: istioctl ## Install istio.
+.PHONY: istioctl-install
+istioctl-install: istioctl ## Install istio.
 	$(ISTIOCTL) operator init
 	kubectl apply -f $(ISTIO_INSTALL_DIR)/istio-operator.yaml
 
-.PHONY: istio-uninstall
-istio-uninstall: istioctl ## Uninstall istio.
+.PHONY: istioctl-uninstall
+istioctl-uninstall: istioctl ## Uninstall istio.
 	$(ISTIOCTL) x uninstall -y --purge
 
-.PHONY: istio-verify-install
-istio-verify-install: istioctl ## Verify istio installation.
+.PHONY: istioctl-verify-install
+istioctl-verify-install: istioctl ## Verify istio installation.
 	$(ISTIOCTL) verify-install -i $(ISTIO_NAMESPACE)
+
+.PHONY: sail-install
+sail-install: kustomize
+	$(MAKE) install-metallb
+	$(KUSTOMIZE) build $(ISTIO_INSTALL_DIR)/sail | kubectl apply -f -
```

Comment:
I see your suggestion, but given we use this for all of our kustomize uses elsewhere I think it would be strange to have this single occurrence use `kubectl apply -k`. Perhaps we should have a follow up issue to replace all `$(KUSTOMIZE) build .. | kubectl apply -f` if we think that makes sense?